test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -73,6 +80,8 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             // Assert
             Assert.Equal(0, result.ExitCode);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
Here's the limitation of binding folder to the test class name, you can't leverage the setup-teardown mechanism of xUnit.